Dockerfile

```diff
@@ -23,11 +26,10 @@ RUN apt-get update && apt-get -yq dist-upgrade && \
 # install ipython
 RUN pip install --no-cache-dir ipython
 
-# copy over files and install requirements
-ADD . /src/pyquil
-WORKDIR /src/pyquil
-RUN pip install -e .
+# install pyquil
+RUN pip install --index-url $primary_index_url --extra-index-url https://pypi.org/simple pyquil==$pyquil_version
```

Comment:
Images get published after PyPI publishing, so we'll just use that (it's a nice way to validate the PyPI publish too).